pganonymizer/utils.py

```diff
@@ -266,7 +266,7 @@ def create_database_dump(filename, db_args):
     :param dict db_args: A dictionary with database related information
     """
     arguments = '-d {dbname} -U {user} -h {host} -p {port}'.format(**db_args)
-    cmd = 'pg_dump -p -Fc -Z 9 {args} -f {filename}'.format(
```

Comment:
Thanks for your pull request. The `-p` flag is indeed a duplicate and is already used for the port option. I changed the target branch to `develoment` as I don't merge directly into the `master`.

Could you please fix the unit test that asserts for the correct arguments for `pg_dump`? Despite the test, your change looks good to me.